Vagrantfile

```diff
@@ -65,7 +65,13 @@ Vagrant.configure(2) do |config|
 
     # Change grub to support cgroups memory if necessary
     config.vm.provision "shell", path: "cookbook/system-config/grub-enable-cgroup-memory.sh"
-    config.vm.provision :reload
+    fileCheck = lambda {
+        if File.exists?("VAGRANT_NEEDS_RELOAD")
+            File.delete("VAGRANT_NEEDS_RELOAD")
+            return true
+        end
```

Comment:
Yeah, that would be reasonable. The plugin checks explicitly for true, so anything else is considered false, but it is still good practice to return false I think.